prime-router/src/testIntegration/kotlin/datatests/mappinginventory/xcn/XCNtoPractitionerTests.kt

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.prime.router.datatests.mappinginventory.xcn
+
+import gov.cdc.prime.router.datatests.mappinginventory.verifyHL7ToFHIRToHL7Mapping
+import org.junit.jupiter.api.Test
+
+class XCNtoPractitionerTests {
+    @Test
+    fun `test translate to HL7 to FHIR to HL7`() {
+        assert(verifyHL7ToFHIRToHL7Mapping("xcn/xcn-to-practitioner-xcn19-20-populated-xcn17-empty").passed)
+        assert(verifyHL7ToFHIRToHL7Mapping("xcn/xcn-to-practitioner-xcn17-populated-xcn19-20-empty").passed)
+        assert(verifyHL7ToFHIRToHL7Mapping("xcn/xcn-to-practitioner-uuid-assigning-authority").passed)
+        assert(verifyHL7ToFHIRToHL7Mapping("xcn/xcn-to-practitioner-uuid-assigning-authority-no-namespace").passed)
+        assert(verifyHL7ToFHIRToHL7Mapping("xcn/xcn-to-practitioner-iso-assigning-authority-no-namespace").passed)
```

Comment:
Can we add test for a different value for HD.3 like DNS or CLIA to verify that the organization gets created?